customer/src/main/java/com/yas/customer/model/UserAddress.java

```diff
@@ -0,0 +1,49 @@
+package com.yas.customer.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "user_address")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class UserAddress {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String userId;
+
+    private Long addressId;
+
+    private Boolean isActive;
+
+    @CreationTimestamp
+    private ZonedDateTime createdOn;
+
+    @CreatedBy
+    private String createdBy;
+
+    @UpdateTimestamp
+    private ZonedDateTime lastModifiedOn;
+
+    @LastModifiedBy
```

Comment:
@CreationTimestamp ,@CreatedBy, @UpdateTimestamp, @LastModifiedBy. These annotation, they will no longer have any effect since there is no listener to handle these annotations. Add annotation @EntityListeners(CustomAuditingEntityListener.class) 
Reference
![image](https://user-images.githubusercontent.com/62151844/229430529-aa3d7c9c-730b-40e2-9ae3-f507b1433664.png)

Reply:
thank Lam